src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/CurrentProjectSelectionStep.cs

```diff
@@ -17,6 +17,7 @@ public class CurrentProjectSelectionStep : UpgradeStep
         private readonly IUserInput _input;
         private readonly ITargetFrameworkMonikerComparer _tfmComparer;
         private readonly ITargetTFMSelector _tfmSelector;
+        private IProject[]? _orderedProjects;
```

Comment:
Should we set this to null after applying? Or on reset?

Reply:
That's an interesting question. On principle, I think you're right that this should probably be set to null on reset (since reset 'undoes' initialization). In practice, I don't think it matters since the project order shouldn't change.

Still, it's probably better to be correct in case anything changes in the future.